modules/basegl/src/processors/instancerenderer.cpp

```diff
@@ -269,7 +280,32 @@ InstanceRenderer::InstanceRenderer()
         types can be added.)"_unindentHelp,
              prefabs()}
     , vecPorts_{}
-    , camera_("camera", "Camera", util::boundingBox(inport_))
+    , camera_("camera", "Camera",
+              [this]() -> std::optional<mat4> {
+                  if (numberOfvertices_ > 0) {
+                      rendercontext::activateDefault();
+                      feedbackBuffer_.bindBase(0);
```

Comment:
I think that it would make more sense, and make the code cleaner, if we could use compute shaders for this. Adding transform feedback to the rendering will impact the performance negatively?

Reply:
only calculate it on demand now... transform feedback works on macOS compute shaders does not...